lib/publiccloud/provider.pm

```diff
@@ -295,7 +335,7 @@ sub terraform_apply {
     record_info('TFM cmd', $cmd);
 
     assert_script_run($cmd);
-    my $ret = script_run('terraform apply -no-color myplan', TERRAFORM_TIMEOUT);
+    my $ret = script_run('terraform apply -no-color -auto-approve myplan', TERRAFORM_TIMEOUT);
```

Comment:
From https://learn.hashicorp.com/terraform/development/running-terraform-in-automation
I would go with 
```suggestion
    my $ret = script_run('terraform apply -no-color -input=false myplan', TERRAFORM_TIMEOUT);
```

Reply:
OK, will do this.